docs/source/gmaps.rst

```diff
@@ -221,9 +221,11 @@ Markers are currently limited to the Google maps style drop icon. If you need to
 
     gmaps.configure(api_key="AI...")
 
-    starbucks_locations = gmaps.datasets.load_dataset("starbucks_uk")
+    df = gmaps.datasets.load_dataset_as_df("starbucks_kfc_uk")
+    starbucks_df = df.query("chain_name == 'starbucks'").drop("chain_name", axis=1)
```

Comment:
The `.query` notation is neat -- I didn't know you could do that. Do you think it's more readable or more common than `df[df["chain_name"] == "starbucks"]`?

To get rid of the `chain_name` column, for consistency with the rest of the documentation (see e.g. the snippet for weighted [heatmaps](http://jupyter-gmaps.readthedocs.io/en/latest/gmaps.html#weighted-heatmaps)) and for robustness, I would prefer sub-selecting the right columns in the data frame, rather than dropping `chain_name` ; i.e. prefer 

```
starbucks_df[['latitude', 'longitude']])
```

over `.drop("chain_name", axis=1)`.